index.html

```diff
@@ -5,10 +5,55 @@
         <title>roboberryPi!</title>
         <link rel="stylesheet" href="assets/roboStyles.css">
         <script src="assets/roboScripts.js"></script>
-        <link type="image/png" sizes="32x32" rel="icon" href="assets/raspberry-pi.png">
+        <link type="image/png" sizes="32x32" rel="icon" href="https://cdn-icons-png.flaticon.com/32/5969/5969184.png">
```

Comment:
Changed to hyperlink images because then I didn't have to serve the images themselves